TeamZeroServer/src/server/ServerMain.java

```diff
@@ -125,8 +125,26 @@ public void onMessage(WebSocket websocket, String message) {
 			JSONObject json = new JSONObject(message);
 			String msgType = json.getString(JSON_KEY_MESSAGE_TYPE);
 			if (msgType == CASE_LOGIN) {
```

Comment:
@ie208 can you please use .equals here instead of ==

Reply:
done

Reply:
Ok, I'll merge.